frontend/pages/recipe/create/url.vue

```diff
@@ -114,19 +115,32 @@ export default defineComponent({
       },
     });
 
+    const stayInEditMode = computed({
+      get() {
+        return route.value.query.stay_in_edit_mode === "1";
+      },
+      set(keepEditMode: boolean) {
+        let stay_in_edit_mode = "0";
+        if (keepEditMode) {
+          stay_in_edit_mode = "1";
+        }
+        router.replace({ query: { ...route.value.query, stay_in_edit_mode } });
+      },
+    });
+
```

Comment:
If you want a default value to be true, you have to initialize it when the component mounts otherwise it will be falsey by default.